starter-code/index.js

```diff
@@ -1,17 +1,54 @@
 class SortedList {
-  constructor() {}
+  constructor(items, length) {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.length = this.items.length;
+    this.items.sort((a, b) => a - b);
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (this.length < pos) {
+      throw new Error("OutOfBounds");
+    } else {
+      return this.items[pos];
+    }
+  }
 
-  max() {}
+  max() {
+    if (this.length <= 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return Math.max(...this.items);
+    }
+  }
 
-  min() {}
+  min() {
+    if (this.length <= 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return Math.min(...this.items);
+    }
+  }
 
-  sum() {}
+  sum() {
+    if (this.length <= 0) {
+      return 0;
+    } else {
+      return this.items.reduce((accu, value) => accu + value);
+    }
+  }
 
-  avg() {}
+  avg() {
+    if (this.length <= 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return this.items.reduce((accu, value) => accu + value) / this.length;
```

Comment:
Keep in mind that you can use the function: this.sum() so you don't have to reduce twice